app/src/main/java/org/fossasia/openevent/general/ticket/TicketDetailsViewHolder.kt

```diff
@@ -20,14 +20,15 @@ class TicketDetailsViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView
                 itemView.price.text = resource.getString(R.string.donation)
                 itemView.subTotal.text = "$eventCurrency${"%.2f".format(donationAmount)}"
             }
-            TICKET_TYPE_FREE -> {
```

Comment:
Why remove this?

Reply:
Because it was not being used.
For both free and paid tickets, TICKET_TYPE_PAID is used.
Do you want me to add the TICKET_TYPE_FREE back?

Reply:
They were being used, if free tickets are also of type PAID, then it is a bug in server

Reply:
Ok